core/chains/evm/gas/helpers_test.go

```diff
@@ -157,6 +157,7 @@ type MockGasEstimatorConfig struct {
 	FeeCapDefaultF      *assets.Wei
 	LimitMaxF           uint64
 	ModeF               string
+	EstimateGasLimitF   bool
```

Comment:
Think we do this because the linter doesn't like it when the field has the same name as the method to access it. Not sure why "F" but just following the pattern here